index.js

```diff
@@ -53,10 +53,12 @@ function loremPicsum(opts) {
     throw new Error("opts.width must be a number");
   }
 
-  if (opts.grayscale) {
-    path.push("g");
-  }
-
+    // seed - ignored if image id set
+    if (opts.seed && typeof opts.image === 'undefined') {
+      path.push("seed");
+      path.push(opts.seed);
```

Comment:
`path.push(opts.seed.toString());`

would be better to ensure the same behaviour as width and height. Generally everything pushed into the path should be cast as a string.